scripts/ioctool

```diff
@@ -110,12 +256,48 @@ elif [ "$CMD" == "less" ]; then
 
 #################################################################
 
+elif [ "$CMD" == "cat" ]; then
+   ioccfg "$NAME"
+   cat "$iocfile"
+
+#################################################################
+
 elif [ "$CMD" == "data" ]; then
    iocdatapath=/reg/d/iocData/"${NAME}"/iocInfo
    if [ -d "$iocdatapath" ]; then
       echo "${iocdatapath}"
    else
-      echo "$iocdatapath could not be found. Exiting..."
+      echo "$iocdatapath could not be found. Exiting..." >&2
+   fi
+
+#################################################################
+
+elif [ "$CMD" == "autosave" ]; then
+   autosavefile=/reg/d/iocData/"${NAME}"/autosave/${NAME}.sav
```

Comment:
This branch does not work for the pytmc-generated twincat IOC autosave files because the file names are different:
```
$ ls /cds/data/iocData/ioc-lfe-motion/autosave/
info_positions.req                info_positions.sav_210325-173832  info_positions.sav_230119-152437  info_settings.sav_201118-210022  info_settings.sav_210823-132817
info_positions.sav                info_positions.sav_210325-175539  info_positions.sav_230121-060135  info_settings.sav_210106-165713  info_settings.sav_210918-133200
info_positions.sav0               info_positions.sav_210326-165516  info_positions.sav_230121-123032  info_settings.sav_210115-133120  info_settings.sav_211005-100349
info_positions.sav1               info_positions.sav_210402-105142  info_positions.sav_230204-164422  info_settings.sav_210119-153527  info_settings.sav_211012-101144
info_positions.sav2               info_positions.sav_210423-095513  info_positions.sav_230220-064720  info_settings.sav_210211-110618  info_settings.sav_211021-102332
```

Reply:
Will make a pull request for this issue.